### PR TITLE
Fix NPE and wrong formatting options:

### PR DESCRIPTION
**Pull request by jbkt**
- NullPointerException when no parameters in Maven configuration.
- All files in current directory formatted instead of all files
  in current project (it's a problem on a multi-module projects or when you run maven from another directory).

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <groupId>org.antipathy</groupId>
     <artifactId>mvn-scalafmt</artifactId>
     <packaging>maven-plugin</packaging>
-    <version>0.3_${version.scalafmt}</version>
+    <version>0.4_${version.scalafmt}-SNAPSHOT</version>
```
Comment (SimonJPegg) [inline pom.xml]: Could you remove the SNAPSHOT tag?

File: src/main/java/org/antipathy/scalafmtmvn/FormatMojo.java [hunk 2]
```diff
@@ -5,31 +5,43 @@
...
 @Mojo(name = "format")
 public class FormatMojo extends AbstractMojo {
 
-    @Parameter(property="format.configLocation",defaultValue="")
+    @Parameter(property = "format.configLocation")
     private String configLocation;
-    @Parameter(property="format.parameters",defaultValue="")
+    @Parameter(property = "format.parameters")
     private String parameters;
+    @Parameter(defaultValue = "${project}", required = true, readonly = true)
```
Comment (SimonJPegg) [inline src/main/java/org/antipathy/scalafmtmvn/FormatMojo.java:22]: Could you update the readme to include this new param?

File: src/main/scala/org/antipathy/scalafmtmvn/Formatter.scala [hunk 3]
```diff
@@ -15,16 +14,19 @@ object Formatter {
    * Gets calls scalafmt with the config file specified at configLocation
    * @param configLocation the location of a scalafmt.conf file
    */
-  def format(configLocation: String, parameters: String): Unit = {
+  def format(configLocation: String, parameters: String, sourceLocations: Array[String]): Unit = {
```
Comment (SimonJPegg) [inline src/main/scala/org/antipathy/scalafmtmvn/Formatter.scala:17]: Could you also update the unit test with the new argument?

**Review by SimonJPegg — CHANGES_REQUESTED** [review]
Looks good, couple of small comments

File: pom.xml (hunk 1 above)
Comment (jbkt) [reply pom.xml]: Done

File: src/main/java/org/antipathy/scalafmtmvn/FormatMojo.java (hunk 2 above)
Comment (jbkt) [reply src/main/java/org/antipathy/scalafmtmvn/FormatMojo.java:22]: Actually this param is not visible by user, it's a Maven property, but I did update the version and fixed some typos in the README.ms.

File: src/main/scala/org/antipathy/scalafmtmvn/Formatter.scala (hunk 3 above)
Comment (jbkt) [reply src/main/scala/org/antipathy/scalafmtmvn/Formatter.scala:17]: Sure, done.

**Comment by jbkt** [comment]
Thanks for the quick reviews!
When do you think you will be able to release a version?

**Comment by SimonJPegg** [comment]
Already pushed, should appear on mvn central in the next hour or so.

**Comment by jbkt** [comment]
Great 👍

**Pull request merged**